### PR TITLE
Adds delivery report step dates 

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds order/prep and sequencing dates for the samples in the delivery report

How to test:
0. install on stage on rasta 
1. us
1. cg upload delivery-report -p CUST-ID FAMILY-ID

Expected outcome:
order/prep and sequencing dates are present for the samples

### CODE REVIEW

File: tests/meta/report/test_meta_report.py [hunk 1]
```diff
@@ -64,12 +67,14 @@ def test_incorporate_delivery_date_from_lims(lims_samples, report_api):
 
     # WHEN fetch_and_add_delivery_date_from_lims
```
Comment (ingkebil) [inline tests/meta/report/test_meta_report.py]: Comment out of date?

File: tests/meta/report/test_meta_report.py [hunk 2]
```diff
@@ -64,12 +67,14 @@ def test_incorporate_delivery_date_from_lims(lims_samples, report_api):
...
     samples = lims_samples
-    report_api._incorporate_delivery_date_from_lims(samples)
+    report_api._incorporate_dates_from_lims(samples)
 
     # THEN
     # each sample has a property processing_time with a value of the days between received_at and
```
Comment (ingkebil) [inline tests/meta/report/test_meta_report.py]: I don't understand how the comment is related to the test?

**Comment by emiliaol** [comment]
I'm assuming I can test this on hasta as well? I'll give it a go :)

**Comment by emiliaol** [comment]
I need a case I can test this on. Needs to be available in all databases and have analysis results in HK stage.

**Comment by barrystokman** [comment]
Would fancyhorse work?

**Comment by barrystokman** [comment]
From zube:
```
Expected outcome:
It should be possible to see which dates different steps were performed:

    - order date
    - library prep date 
    - sequencing date
```
Other dates are out of scope for this user story.

File: tests/meta/report/test_meta_report.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply tests/meta/report/test_meta_report.py]: fixed

File: tests/meta/report/test_meta_report.py (hunk 2 above)
Comment (patrikgrenfeldt) [reply tests/meta/report/test_meta_report.py]: fixed

**Pull request merged**